TrafficCapture/kafkaPrinter/src/main/java/org/opensearch/migrations/KafkaPrinter.java

```diff
@@ -36,6 +38,15 @@ static class Parameters {
                 names = {"-g", "--group-id"},
                 description = "Client id that should be used when communicating with the Kafka broker.")
         String clientGroupId;
+        @Parameter(required = false,
+                names = {"-m", "--enable-msk-client"},
+                description = "Enables properties required for connecting to an MSK public endpoint.")
+        Boolean isMSKPublic = false;
```

Comment:
nit: I think `boolean` primitive is fine here, don't need the object

Reply:
We may use MSK with an internal endpoint if everything is within the same VPC. I would change this to `mskAuthEnabled`